azurelinuxagent/common/osutil/mariner.py

```diff
@@ -0,0 +1,85 @@
+#
+# Copyright 2018 Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Requires Python 2.6+ and Openssl 1.0+
+#
+
+import errno
+import azurelinuxagent.common.conf as conf
+import azurelinuxagent.common.utils.fileutil as fileutil
+from azurelinuxagent.common.osutil.default import DefaultOSUtil
+from azurelinuxagent.common.exception import OSUtilError
+
+class MarinerOSUtil(DefaultOSUtil):
+    def __init__(self):
+        super(MarinerOSUtil, self).__init__()
+        self.agent_conf_file_path = '/etc/waagent.conf'
```

Comment:
This is unnecessary, `DefaultOSUtil` already specifies the same filepath

Reply:
Updated